modules/sdk/gradle-plugins-service-builder/README.markdown

```diff
@@ -0,0 +1,153 @@
+
+Property Name | Default Value
+------------- | -------------
+[`args`](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.JavaExec.html#org.gradle.api.tasks.JavaExec:args) | Service Builder command line arguments
+[`classpath`](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.JavaExec.html#org.gradle.api.tasks.JavaExec:classpath) | [`project.configurations.serviceBuilder`](#liferay-service-builder-dependency)
+[`main`](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.JavaExec.html#org.gradle.api.tasks.JavaExec:main) | `"com.liferay.portal.tools.service.builder.ServiceBuilder"`
+[`systemProperties`](https://docs.gradle.org/current/dsl/org.gradle.api.tasks.JavaExec.html#org.gradle.api.tasks.JavaExec:systemProperties) | `["file.encoding": "UTF-8"]`
+
+
+#### Task Properties
+
+Property Name | Type | Default Value | Description
+------------- | ---- | ------------- | -----------
+<a name="apidir"></a>`apiDir` | `File` | `null` | The directory where the service API Java source files are generated.
+`autoImportDefaultReferences` | `boolean` | `true` | Whether to automatically add default references like `com.liferay.portal.ClassName`, `com.liferay.portal.Resource` and `com.liferay.portal.User` to the services.
+`autoNamespaceTables` | `boolean` | `true` | Whether to prefix table names by the namespace specified in the `service.xml` file.
+`beanLocatorUtil` | `String` | `"com.liferay.util.bean.PortletBeanLocatorUtil"` | The fully qualified class name of the bean locator class to use in the generated service classes.
+`buildNumber` | `long` | `1` | Force a specific value for the `build.number` property in the `service.properties` file.
+`buildNumberIncrement` | `boolean` | `true` | Whether to automatically increment the `build.number` property in the `service.properties` file by one at every service generation.
+<a name="hbmfile"></a>`hbmFile` | `File` | `null` | The destination of the generated Hibernate Mapping file.
+<a name="impldir"></a>`implDir` | `File` | `null` | The directory where the service Java source files are generated.
+<a name="inputfile"></a>`inputFile` | `File` | `null` | The `service.xml` file of the project.
+`modelHintsConfigs` | `Set` | `["classpath*:META-INF/portal-model-hints.xml", "META-INF/portal-model-hints.xml", "classpath*:META-INF/ext-model-hints.xml", "classpath*:META-INF/portlet-model-hints.xml"]` | The locations of the [model hints](https://dev.liferay.com/develop/tutorials/-/knowledge_base/7-0/customizing-model-entities-with-model-hints) files used by Liferay Service Builder in order to generate the service layer.
+<a name="modelhintsfile"></a>`modelHintsFile` | `File` | `null` | The model hints file of the project.
+<a name="osgimodule"></a>`osgiModule` | `boolean` | `false` | Whether to generate the service layer for OSGi modules.
+<a name="pluginname"></a>`pluginName` | `String` | `null` | If specified, enables additional generation features for non-OSGi modules, like the `Clp` classes.
```

Comment:
Yes! I'm not an expert on SB, I think that's what that argument does :sweat_smile: Maybe @gamerson knows more about it!